lib/resolveConfig.js

```diff
@@ -6,17 +6,7 @@ const createStylelint = require('./createStylelint');
 const getConfigForFile = require('./getConfigForFile');
 
 /**
- * Resolves the effective configuration for a given file. Resolves to `undefined`
- * if no config is found.
- * @param {string} filePath - The path to the file to get the config for.
- * @param {Pick<
- *   import('stylelint').LinterOptions,
- *   | 'cwd'
- *   | 'config'
- *   | 'configBasedir'
- *   | 'configFile'
- * >} options - The options to use when creating the Stylelint instance.
- * @returns {Promise<import('stylelint').Config | undefined>}
+ * @type {import('stylelint').default['resolveConfig']}
```

Comment:
[note] To reduce duplication, I've moved the description in JSDoc to the `.d.ts` file.